.github/workflows/pr-verify.yaml

```diff
@@ -12,8 +12,20 @@ jobs:
     runs-on: ubuntu-latest
     name: verify PR contents
     steps:
-    - name: Verifier action
-      id: verifier
-      uses: kubernetes-sigs/kubebuilder-release-tools@012269a88fa4c034a0acf1ba84c26b195c0dbab4 # tag=v0.4.3
-      with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Verifier action
+        id: verifier
+        uses: kubernetes-sigs/kubebuilder-release-tools@012269a88fa4c034a0acf1ba84c26b195c0dbab4 # tag=v0.4.3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.21
```

Comment:
We should be able to paramaterize this - see https://github.com/kubernetes-sigs/cluster-api/blob/3c86142d92a7dc9bf84f3991731de1c73bd9755c/.github/workflows/pr-golangci-lint.yaml#L25-L26

Reply:
Let's please not add it to this GitHub action.

We already have a `verify` Makefile target. Let's run it there, i.e. let's add it to the list in ~l.684 in Makefile (ALL_VERIFY_CHECKS)